<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/rsc/Icons/calc_icon.png</normaloff>:/rsc/Icons/calc_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(226, 226, 226);
</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>304</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(43, 40, 48);
	font-weight: bold;
	font-size: 12px;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;*Ежемесячный платёж,&lt;br/&gt; рассчитанный по дифференциальному методу, &lt;br/&gt;приведён на последний месяц.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>304</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>340</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 14px;
	border: 2px solid gray;
	border-color: #d09d80;
	border-radius: 15px;
	margin-top: 10px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
	padding: 0px 5px;
}
</string>
   </property>
   <property name="title">
    <string>Введите исходные параметры</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Сумма кредита (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_sum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(43, 40, 48);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Срок кредита (мес.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_period">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(43, 40, 48);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Ставка процента (%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_percent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(43, 40, 48);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="percent_method">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>282</width>
        <height>76</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
	border: 1px solid gray;
	border-radius: 0px
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
}
</string>
      </property>
      <property name="title">
       <string>Метод начисления процентной ставки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="check_annuitet">
         <property name="styleSheet">
          <string notr="true">QRadioButton {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>аннуитетный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="check_different">
         <property name="styleSheet">
          <string notr="true">QRadioButton {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>дифференцированный</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	font-weight: bold;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 87, 104);
	border-radius: 8px;
}
QPushButton:hover {
	background-color: rgb(143, 143, 143);
}
QPushButton:pressed {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>304</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>340</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 14px;
	border: 2px solid gray;
	border-color: #d09d80;
	border-radius: 15px;
	margin-top: 10px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
	padding: 0px 5px;
}
</string>
   </property>
   <property name="title">
    <string>Расчётные данные</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Ежемесячный платёж (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_monthpay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>70</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Переплата по кредиту (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_creditsum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>70</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Общая выплата (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_total">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_calc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	font-weight: bold;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 87, 104);
	border-radius: 8px;
}
QPushButton:hover {
	background-color: rgb(143, 143, 143);
}
QPushButton:pressed {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
